vcs_server/Visualizer.py

```diff
@@ -61,11 +61,10 @@ def plot(self, prevWindowId, varSpecs, method, template, width, height, opts={})
                                 print "Got {} as a transform method".format(method)
             if ('axis_order' in varSpec):
                 indexOrder = varSpec['axis_order']
-                stringOrder = var.getOrder()
-                stringOrder = ''.join([stringOrder[i] for i in indexOrder])
+                axisOrder = var.getAxisIds()
+                stringOrder = ''.join(["({})".format(axisOrder[i]) for i in indexOrder])
                 var = var(order=stringOrder)
```

Comment:
@James-Crean I'm not 100% sure but looks like you're retrieving the variable in the ***exact*** same order as it is stored, hence it's just useless. I think you want 
```python
var=var(order=varSpec["axis_order"])
```

or at the very least construct your ordering based on 'axis_order' of varSpec

Reply:
I was going to say it looked good to me, as `indexOrder` is supposed to be a list of integers giving the desired order (e.g. `[0, 1, 3, 2]`).

Reply:
... and `indexOrder` *does* come from `axis_order` of `varSpec` ...